brdnet/evaluate_models.py

```diff
@@ -0,0 +1,151 @@
+'''Compare the performance of different models in learning to differentiate between healthy
+and unhealthy gene expression
+'''
+import argparse
+import importlib
+import os
+
+# Mute INFO and WARNING logs from tensorflow
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '2'
+
+import pandas
+import pyod
+import sklearn
+
+import classifier
+import models
+import utils
+
+
+def model_from_pyod(model_name):
+    try:
```

Comment:
Very minor, but looks like you're missing the description for the functions in evaluate_models.py

Reply:
Good catch!